src/models/connectivity-threshold.ts

```diff
@@ -0,0 +1,16 @@
+import Joi from 'joi';
```

Comment:
Would have been nice to include a todo and a //deprecated into the IssueTriggerRule model. 
You could check out this review for inspiration: https://github.com/withthegrid/platform-sdk/pull/347